app/helpers/buyers_helper.rb

```diff
@@ -0,0 +1,57 @@
+module BuyersHelper
+  def us_states
+    [
+      ['AK', 'AK'],
```

Comment:
I don't know why I did it like that. I think I followed the way some dude on stack overflow did it. Will amend for next time!